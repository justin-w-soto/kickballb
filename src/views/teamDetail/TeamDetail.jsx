import React from 'react'

export const TeamDetail = () => {
    return (
        <div>
            <h1>Team Detail</h1>
        </div>
    )
}
