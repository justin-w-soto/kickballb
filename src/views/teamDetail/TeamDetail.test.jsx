import { render, screen } from '@testing-library/react'
import { MemoryRouter, Route } from 'react-router-dom'
import { TeamDetail } from './TeamDetail'

it('Should render a list of players to the page', async () => {
    const {container} = render(
        <MemoryRouter >
            <Route path="/teams/:id">
                <TeamDetail />
            </Route>
        </MemoryRouter>
    );

    expect(container).toMatchSnapshot();
})