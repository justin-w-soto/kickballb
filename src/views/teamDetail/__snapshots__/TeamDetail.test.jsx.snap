// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render a list of players to the page 1`] = `<div />`;
